src/IOSDebugLauncher/VcRemoteClient.cs

```diff
@@ -58,7 +58,7 @@ public static VcRemoteClient GetInstance(IOSLaunchOptions options)
         public Launcher.RemotePorts StartDebugListener()
         {
             string remotePortsJsonString;
-            CallVcRemote(new Uri("debug/setupForDebugging?target=" + _launchOptions.IOSDebugTarget.ToString() + "?deviceUdid=" + _launchOptions.DeviceUdid, UriKind.Relative), LauncherResources.Info_StartingDebugListener, out remotePortsJsonString);
+            CallVcRemote(new Uri("debug/setupForDebugging?target=" + _launchOptions.IOSDebugTarget.ToString() + "&deviceUdid=" + _launchOptions.DeviceUdid, UriKind.Relative), LauncherResources.Info_StartingDebugListener, out remotePortsJsonString);
```

Comment:
It doesn't hurt us at all. The parameter will just be empty, and vcremote will default to an empty string if there is no deviceudid parameter.